ci/rtools/aws_c_common_ep.patch

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
```

Comment:
Can we have a comment on the top of patch files when the reason for patching is not obvious?

Reply:
Yeah, that makes sense. I'll add a comment.

Reply:
What do you think about adding a readme to the entire dir since the reason for the patches is related? I did that in f17cb8fa557c3f9de9863b8e89aaf72b45e38358.

Reply:
Thank you @amoeba , yes, that makes sense.